src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+public class Todo extends Entry {
+
+    /**
+     * Constructor for Todo.
+     */
+    Todo() {
+        super();
+    }
+
+    /**
+     * Constructor for Todo.
```

Comment:
Seems like the javadocs description for the parameter is missing for this JavaDocs header comment

Reply:
resolved, thanks!